modules/tinymce/src/plugins/table/test/ts/browser/BorderNormalizationTest.ts

```diff
@@ -0,0 +1,137 @@
+import { describe, context, it } from '@ephox/bedrock-client';
+import { Arr } from '@ephox/katamari';
+import { Attribute, SugarElement } from '@ephox/sugar';
+import { assert } from 'chai';
+import * as BorderNormalizer from 'tinymce/plugins/table//core/BorderNormalizer';
+import { DomModifier } from 'tinymce/plugins/table/ui/DomModifier';
+
+type normalizer = (modifier: DomModifier, cell: SugarElement<HTMLTableCellElement>, color: string) => void;
```

Comment:
```suggestion
type Normalizer = (modifier: DomModifier, cell: SugarElement<HTMLTableCellElement>, color: string) => void;
```

Reply:
Oops. Been writing reason for too long there everything is lower case first.